conf/configuration.go

```diff
@@ -81,20 +81,17 @@ type Configuration struct {
 }
 
 func loadEnvironment(filename string) error {
-	// default to .env if an empty string is passed as a filename
-	// godotenv will only default if passed an empty argument list
-	filenames := []string{}
+	var err error
 	if filename != "" {
-		filenames = append(filenames, filename)
-	}
-	if err := godotenv.Load(filenames...); err != nil {
+		err = godotenv.Load(filename)
+	} else {
+		err = godotenv.Load()
 		// handle if .env file does not exist, this is OK
-		if os.IsNotExist(err) && filename == "" {
+		if os.IsNotExist(err) {
```

Comment:
I think that we _should_ return an error when you specify a filename. It would be odd to specify a filename and then be silently configured out of the env.